clib/memprof_coq.mli

```diff
@@ -0,0 +1,53 @@
+(* From memprof-limits *)
+val is_interrupted : unit -> bool
+
+module Masking : sig
+
+  val with_resource :
+    acquire:('a -> 'b) -> 'a -> scope:('b -> 'c) -> release:('b -> unit) -> 'c
+
```

Comment:
I'm happy to update the docs here from upstream, basically this runs `acquire` and `release` with async exns masked.